RadixWallet/Clients/DeviceFactorSourceClient/DeviceFactorSourceClient+Live.swift

```diff
@@ -80,7 +80,7 @@ extension DeviceFactorSourceClient: DependencyKey {
 
 		@Sendable
 		func factorSourcesMnemonicPresence() async -> AnyAsyncSequence<[FactorSourceHasMnemonic]> {
-			await combineLatest(profileStore.factorSourcesValues(), secureStorageClient.keychainChanged())
+			await combineLatest(profileStore.factorSourcesValues(), secureStorageClient.keychainChanged().prepend(()))
```

Comment:
Given on app start we don't have a keychainChanged event, we weren't getting this fired